src/api/cli/create_rawtx.go

```diff
@@ -464,15 +464,17 @@ func getSufficientUnspents(unspents []UnspentOut, amt uint64) ([]UnspentOut, err
 // NewTransaction create skycoin transaction.
 func NewTransaction(utxos []UnspentOut, keys []cipher.SecKey, outs []coin.TransactionOutput) (*coin.Transaction, error) {
 	tx := coin.Transaction{}
-	// keys := make([]cipher.SecKey, len(utxos))
 	for _, u := range utxos {
 		tx.PushInput(cipher.MustSHA256FromHex(u.Hash))
 	}
 
 	for _, o := range outs {
+		if o.Coins%1e5 != 0 {
```

Comment:
`1e5` should be in a const variable at the top of the file or imported from a single place in a skycoin/src/ package